src/enums/lease.ts

```diff
@@ -38,16 +38,21 @@ export enum OccupantRelationship {
   SIBLING = 'Sibling',
   GRAND_PARENT = 'Grand Parent',
   GRAND_CHILD = 'Grand Child',
-  OTHER = 'Other'
+  OTHER = 'Other',
 }
 
 export enum LeaseAgreementSource {
-  REALHAUS = "realhaus",
-  NON_REALHAUS = "non-realhaus"
+  REALHAUS = 'realhaus',
+  NON_REALHAUS = 'non-realhaus',
 }
 
 export enum ListingStatus {
-  DEACTIVATED = "Deactivated",
-  UNAVAILABLE = "Unavailable",
-  VACANT = "Vacant"
-}
\ No newline at end of file
+  DEACTIVATED = 'Deactivated',
+  UNAVAILABLE = 'Unavailable',
+  VACANT = 'Vacant',
+}
+
+export enum RentDueDay {
+  FIRST_OF_MONTH = 'First of month',
```

Comment:
Your enum values should not contain spaces. See `FeePaymentFrequency` for a properly defined enum